### PR TITLE
Fixes indentation problem on multiline yaml keys.

### PR DESCRIPTION
**Pull request by erenatas**

### CODE REVIEW

**Comment by romanek-adam** [comment]
I'm not sure we discussed about this - is it possible to create a test for the fix?

**Comment by erenatas** [comment]
> I'm not sure we discussed about this - is it possible to create a test for the fix?

I actually want to write a test case for this, but I'm not sure what kind of test I can write this. I am open to any suggestions.

**Comment by romanek-adam** [comment]
I think just a simple XML with shell builder, with some indentation. Then convert to YAML, parse the YAML file and compare with the expected value - just as you did in the test case for combination-filter. Would that be OK?

**Comment by erenatas** [comment]
> I think just a simple XML with shell builder, with some indentation. Then convert to YAML, parse the YAML file and compare with the expected value - just as you did in the test case for combination-filter. Would that be OK?

Yup, will do that.

File: tests/test_jjb.py [hunk 1]
```diff
@@ -31,3 +43,7 @@ def test_calamari_clients(self):
 
     def test_non_ascii(self):
         self.run_jjb('non-ascii', 'テスト')
+
+    def test_implementation(self):
```
Comment (romanek-adam) [inline tests/test_jjb.py]: Was it meant to be named 'implementation' instead of 'indentation'?

**Review by romanek-adam — APPROVED** [review]

**Review by vdanaila — APPROVED** [review]

**Pull request merged**